app/graphql_server/main.go

```diff
@@ -19,7 +23,81 @@ import (
 )
 
 func main() {
```

Comment:
Is there a convention on where the `main()` func goes? I'm used to seeing it at the bottom of `main.go`, but I don't know if that's an actual convention, or just my brain drawing patterns.

Reply:
The way I've always done it is `main()` at the top, and all other functions below it in the order they are called. Fine to change it if we want something different.

Reply:
Eh, I'm indifferent then. Whatever we settle on we should probably put into Contributing.md though. Maybe get opinions from others on the backend?

Reply:
Couldn't care any less even if I tried.